### ISSUE TITLE
What's the difference between github dark and dark dimmed?  It seems that their background are same

### ISSUE THREAD

**Issue by WindSoilder** [primer/github-vscode-theme#190]

**Comment by dominicegginton** [comment]
TLDR: `GitHub Dark` is an old theme, `GitHub Dark Dimmed` is the new theme that aims for consistency with **GitHub's** Primer design system. 

Hi, @WindSoilder thank you for the question. Maybe we can make this clearer in the README. Here is a list of themes that are available when installing this VSCode extension:

- `GitHub Light`: A legacy old theme.
- `GitHub Dark`: A legacy old theme.
- `GitHub Ligh Default` :sparkles: new :sparkles:: A new theme that aims for consistency with **GitHub's** Primer design system.
- `GitHub Dark Default` :sparkles: new :sparkles:: A new theme that aims for consistency with **GitHub's** Primer design system.
- `GitHub Dark Dimmed` :sparkles: new :sparkles:: A new theme that aims for consistency with **GitHub's** Primer design system.

With the release of [**GitHub** Codespaces](https://github.com/features/codespaces) I believe it was important for the community and **GitHub** to have a constant theme across their interfaces. Hence the work to release the new themes that provide devs with an easy transition from reading and reviewing code on GitHub to working in their IDE.

We decided to keep `GitHub Light` & `GitHub Dark` as many users of the theme didn't like the change to the new light and dark themes implemented with this change. You can read about this [in this discussion](https://github.com/primer/github-vscode-theme/discussions/114) and [pull request](https://github.com/primer/github-vscode-theme/pull/132). This decision enables users who still want to use the old themes to update their extensions and receive bug fixes when nessasery.

**Comment by WindSoilder** [comment]
Thanks for your response :-D

> Maybe we can make this clearer in the README

Yeah I think so, I have created a pr for this, hope that it can help
